dev/com.ibm.ws.cdi.visibility_fat/publish/servers/cdi12BasicServer/server.xml

```diff
@@ -5,6 +5,7 @@
     <featureManager>
         <feature>osgiconsole-1.0</feature>
         <feature>cdi-1.2</feature>
+        <feature>concurrent-1.0</feature> <!-- This is here to enable a liberty feature which can see all application beans, and ensure that doesn't break when a library with a bean is in two different wars -->
```

Comment:
Double check this does enable the auto-feature which causes the problem. Should see tests fail if you revert your product changes.